scripts/Creative.zs

```diff
@@ -62,26 +64,79 @@ var twilightForestMasterTrophy = <simple_trophies:trophy>.withTag({
   TrophyVariant:"gold"
 });
 
+
```

Comment:
Could you please explain what this is?

Reply:
This is changing Gas Trophy.

1. When you watching recipe (R key) you see scrolling gases of many kinds
2. Also you see description that gas trophy crafted with 9 different gases
3. When you place 9 gas tanks in grid crafting function checks for all gases would be different
4. If gases different, it shows trophy as output

There is example how its looks:
https://i.imgur.com/NBFQpsK.gif